.mailmap

```diff
@@ -10,110 +10,88 @@
-Matthias Maier <tamiko@43-1.org> <tamiko+GITHUB@kyomu.43-1.org>
-Matthias Maier <tamiko@43-1.org> <matthias.maier@iwr.uni-heidelberg.de>
-Matthias Maier <tamiko@43-1.org> <maier@tamu.edu>
-
-# Manu Jayadharan
-Manu Jayadharan <maj136@pitt.edu> <35903705+mjayadharan@users.noreply.github.com>
-
-# aggarwal-ashna <53831151+aggarwal-ashna@users.noreply.github.com>
-Ashna Aggarwal <aaggarwal01@ucla.edu> <53831151+aggarwal-ashna@users.noreply.github.com>
-
-# LiMaIG <lmig@math.umd.edu>
-Lise-Marie Imbert-Gerard <lmig@math.umd.edu>
-Lise-Marie Imbert-Gerard <lmig@math.umd.edu> <imbertgerard@cims.nyu.edu>
-
-# agrayver <agrayver@erdw.ethz.ch>
+Vishal Boddu <vishal.boddu@fau.de>
+Benjamin Brands <benjamin.brands@fau.de>
+Ángel R. Carballo <angel.rcarballo@udc.es>
+Fabian Castelli <fabian.castelli@kit.edu> <50630942+gfcas@users.noreply.github.com>
+Nicola Cavallini <nicola.cavallini@sissa.it>
+Nicola Cavallini <nicola.cavallini@unipv.it>
+Praveen Chandrashekar <cpraveen@gmail.com>
+Thomas C. Clevenger <tcleven@clemson.edu>
+Sam Cox <spc29@le.ac.uk>
+Marc Fehling <marc.fehling@gmx.net>
+Rebecca Fildes <rfildes@ucdavis.edu>
+Menno Fraters <menno.fraters@hotmail.com>
+Menno Fraters <menno.fraters@outlook.com>
+Nicola Giuliani <ngiuliani@sissa.it> <nicolagiuliani@dhcp-129-199.sissa.it>
+Anne Glerum <a.c.glerum@uu.nl>
+Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk> <krishnakumarg1984@users.noreply.github.com>
+Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk> <krishnak@vt.edu>
 Alexander Grayver <agrayver@erdw.ethz.ch>
 Alexander Grayver <agrayver@erdw.ethz.ch> <agrayver@agrayver-VirtualBox.(none)>
-
-#
-Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk>
-Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk> <krishnak@vt.edu>
-Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk> <krishnakumarg1984@users.noreply.github.com>
-
-#
-Menno Fraters <menno.fraters@outlook.com>
-
-#
-Ignacio Tomas <itomas@sandia.gov>
-
-#
+Joshua Hanophy <josh.hanophy@gmail.com>
+Mike Harmon <mdh266@gmail.com>
+Graham Harper <grahambenharper@gmail.com>
 Nicole Hayes <hayesni4@msu.edu>
-
-#
-Buğrahan Temür <bugrahan.temuer@tum.de> <bugrahan.temur@gmail.com>
-
-#
-Daniel Arndt <arndtd@ornl.gov> <6da@ornl.gov>
-
-#
+Johannes Heinz <johannes.heinz@tuwien.ac.at> <43043310+jh66637@users.noreply.github.com>
+Dakshina Ilangovan <dakshinai@tamu.edu> <dakshina.ilangova@0785d39b-7218-0410-832d-ea1e28bc413d>
+Lise-Marie Imbert-Gerard <lmig@math.umd.edu>
+Lise-Marie Imbert-Gerard <lmig@math.umd.edu> <imbertgerard@cims.nyu.edu>
+Vladimir Ivannikov <vladimir.ivannikov@hzg.de> <vovannikov@yandex.ru>
+Manu Jayadharan <maj136@pitt.edu> <35903705+mjayadharan@users.noreply.github.com>
+Daniel Jodlbauer <pmnotxor@gmx.at> <jdschule@gmx.at>
+Eldar Khattatov <eld.khattatov@gmail.com>
+Eldar Khattatov <elk58@pitt.edu>
+Sebastian Kinnewig <sebastian@kinnewig.org> <35812030+kinnewig@users.noreply.github.com>
+Sebastian Kinnewig <sebastian@kinnewig.org> <Sebastian@Kinnewig.org>
```

Comment:
```suggestion
```
new/correct mail is identical to old/incorrect mail

Reply:
I kept this one so that the e-mail is all lowercase.